Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -722,8 +723,11 @@ def get_graph_access_token() -> str:
     tenant_id = integration_context.get('tenant_id')
     if not tenant_id:
         raise ValueError(
-            'Did not receive tenant ID from Microsoft Teams, verify the messaging endpoint is configured correctly. '
-            'See https://xsoar.pan.dev/docs/reference/integrations/microsoft-teams#troubleshooting for more information'
+            'Did not receive a tenant ID from Microsoft Teams. Verify that the messaging endpoint in the Demisto bot '
```

Comment:
i believe we're trying to avoid using the "Demisto" word. consult with @nkanon

Reply:
I believe so too... but we have an open ticket to replace all existing references of the 'Demisto Bot' [(CIAC-11613)](https://jira-dc.paloaltonetworks.com/browse/CIAC-11613). In the meantime, I believe it’s better to keep the name as it is to avoid any inconsistencies across different parts of the documentation.